composer.json

```diff
@@ -11,10 +11,13 @@
         }
     ],
     "require": {
-        "php": ">=7.0",
-        "symfony/intl": "~2.6|~3.4|~4.0|~5.0|~6.0|~7.0|~8.0"
+        "php": ">=7.0"
+    },
+    "replace": {
+        "symfony/intl": "*"
     },
     "suggest": {
+        "symfony/intl": "~2.6|~3.4|~4.0|~5.0|~6.0|~7.0|~8.0",
```

Comment:
The right side of `suggest` is supposed to e a human-readable text (see https://getcomposer.org/doc/04-schema.md#suggest) which explains to the user under which circumstances they should install the suggested package